packages/tradershub/src/components/ModalStepWrapper/ModalStepWrapper.scss

```diff
@@ -1,4 +1,4 @@
-.wallets-modal-step-wrapper {
+.modal-step-wrapper {
```

Comment:
I noticed @thisyahlen-deriv suggests to use `Modal` component instead of this `ModalStepWrapper` component, but anyway I want to ask about class names, maybe would be better to use prefix `tradershub` for this package to avoid issues with the existed styles as we did for wallets package? @mahdiyeh-deriv What do you think?

Reply:
We should be using tailwind in this package

Reply:
ModalStepWrapper is going to be removed soon in our package

Reply:
You are right! My bad 😯